atomic_reactor/constants.py

```diff
@@ -60,3 +60,14 @@
 )
 
 DEFAULT_DOWNLOAD_BLOCK_SIZE = 10 * 1024 * 1024 # 10Mb
+
+LABEL_ALIASES = {
+    "name" : ('name', 'Name'),
```

Comment:
If this is being changed, let's just make sure code using it is very clear to read.

With your dict for instance, how should code wanting the label for the repository name look?